content/docs/user-guide/programming/messaging/overview.md

```diff
@@ -0,0 +1,24 @@
+---
+linktitle: Overview
+title: Overview of Event Messaging Systems
+description: Use event buses (EBus), AZ::Interface, and AZ::Event to dispatch messages between systems in Open 3D Engine (O3DE). 
+weight: 100
+---
+
+Event messaging systems are essential to a game engine because they allow each system to communicate with each other. **Open 3D Engine (O3DE)** uses a few types of communication methods, which are separated into the following modules: 
```

Comment:
```suggestion
Event messaging systems are essential to a game engine because they allow systems to communicate with each other. **Open 3D Engine (O3DE)** uses a few types of communication methods, which are separated into the following modules: 
```